src/librustc/dep_graph/dep_node.rs

```diff
@@ -618,6 +618,8 @@ define_dep_nodes!( <'tcx>
     [input] CrateName(CrateNum),
     [] ItemChildren(DefId),
     [] ExternModStmtCnum(DefId),
+    [input] GetLibFeatures,
```

Comment:
This doesn't need to be an input node since it's computed from the HIR.

Reply:
It should be marked as `eval_always` though, since it's accessing `tcx.hir.krate()` anyway. That will make dep-tracking a bit cheaper without compromising accuracy in practice. 

If you want to do some cleanup, you can also do the same for `GetLangItems` (i.e. remove the `with_ignore` and make it `eval_always` instead of `input`). It's OK to do this in a separate PR though.